src/background.js

```diff
@@ -133,18 +133,23 @@ const createModal = async (arg, modalEvent) => {
       let target = arg.target;
       modalRequests[id]['target'] = target;
 
-      let isBlockedAccount = arg.isBlockedAccount;
-      if (isBlockedAccount) {
-        modalRequests[id]['warning'] = true;
-        targetURL += isBlockedAccount ? `&warning=blockedAccount` : `&warning=serverError`;
-      }
-
       targetURL += `&chain=${encodeURIComponent(chain)}`;
       targetURL += `&accountName=${encodeURIComponent(accountName)}`;
       targetURL += `&target=${encodeURIComponent(target)}`;
       targetURL += `&toSend=${encodeURIComponent(toSend)}`;
     }
 
+    if ([Actions.INJECTED_CALL, Actions.TRANSFER].includes(type)) {
```

Comment:
Why only for injection and transfer? Before you were acting on existance of arg.isBlockedAccount, isn't that better?

Reply:
True, I was thinking of limiting it to just transfer and injected_call, but I suppose by removing this if statement it'd be safely checking every prompt.

I was just thinking that it'd be unlikely that a blocked account could do any harm through signedMessage, signNFT, vote, but rather within the transfer or injectedcall functions.

Applying the check to all prompts sounds alright to me, if a blocked user was using the app it'd be funny to always warn them about their own account 😅